cobalt/media/progressive/demuxer_extension_wrapper.cc

```diff
@@ -746,7 +746,7 @@ void DemuxerExtensionWrapper::Request(DemuxerStream::Type type) {
 
   auto& stream =
       (type == DemuxerStream::AUDIO) ? *audio_stream_ : *video_stream_;
-  bool& eos_status =
+  bool eos_status =
```

Comment:
`eos_status` is assigned below, so the reference is required here.  However, I'd suggest to use a pointer instead to be explicit (and in sync with our general style that a pointer is for modification).

Reply:
The change has been reverted.